.github/workflows/go.yml

```diff
@@ -20,3 +20,5 @@ jobs:
         run: go build -v ./...
       - name: Test
         run: go test -v ./...
+      - name: Mod Tidy
+        run: go mod tidy && git diff --exit-code -- go.mod go.sum || (echo "go modules are not tidy, run 'go mod tidy'." && exit 1)
```

Comment:
I copied the implementation from https://github.com/kubeflow/pipelines/blob/32004ab214478451c875ba3dd520ea730503f3bf/test/presubmit-backend-test.sh#L33-L34, but I cannot trigger github actions before your review to verify this works. Can you give me some sort of permission to let me run github actions automatically?

Reply:
Ohh, it just takes some time. Looks like it's working proplery.